package.json

```diff
@@ -1,32 +1,36 @@
 {
-  "name": "nanomsg",
-  "version": "0.4.2",
-  "description": "Node bindings for nanomsg",
-  "main": "lib/index.js",
+  "name":               "nanomsg",
+  "version":            "0.4.2",
+  "description":        "Node bindings for nanomsg",
+  "main":               "lib/index.js",
   "dependencies": {
-    "bindings": "1.1.1",
-    "nan": "~1.6.2"
+    "bindings":         "1.2.1",
+    "nan":              "~1.7.0"
```

Comment:
ya we should try to preserve empirical history that could explain unknowns down the road. 

so definitely, git-blame is prima facie > subjective discrepancies.  i'll revert that part